doc/man1/openssl-fipsinstall.pod.in

```diff
@@ -163,6 +164,11 @@ fails as described above.
 
 Configure the module to not perform run-time security checks as described above.
 
+=item B<-self_test_onload>
+
+Configure the self tests KATS to always run when the module is loaded. If this
```

Comment:
The combination of the word "always" here and "it does this by not writing out the install status to the fips config file" seems risky to me -- it's only "always" until the install status is written by some other mechanism to the fips config file, right?  Should this manual mention this mechanism to change the "always" behavior at some future time?

Reply:
What about..

Do not write the fields related to the "test status indicator" and
"MAC status indicator" to the configuration file. Without these fields the
self tests KATS will run each time the module is loaded. This could be used for
cross compiling since the self tests need to run at least once on the target
machines. Once the self tests have run on the target machine the user could
possibly then add the 2 fields into the configuration using some other mechanism.

Suggest other text if you dont like it...

Reply:
That's quite nice, thank you.
I am not sure that we refer to "the user" much in the existing documentation; if no, we could perhaps end with "run on the target machine, the two fields could potentially be added to the configuration file using some other mechanism", but what you currently have will work just fine.